packages/botonic-nlp/src/tasks/text-classification/storage/config-storage.ts

```diff
@@ -0,0 +1,20 @@
+import { mkdirSync, writeFileSync } from 'fs'
+import { join } from 'path'
+
+import { readJSON } from '../../../utils/file-utils'
+import { TextClassificationConfig } from './types'
+
+export class TextClassificationConfigStorage {
+  public readonly CONFIG_FILENAME = 'config.json'
+
+  load(path: string): TextClassificationConfig {
```

Comment:
when we see how the class is used, we'll decide if we want to move the path to the constructor

Reply:
makes sense! 👍🏽